bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_rooabspdf.py

```diff
@@ -32,56 +32,123 @@ class RooAbsPdf(RooAbsReal):
         "RooAbsPdf::fitTo(RooAbsData&, const RooCmdArg&, const RooCmdArg&, const RooCmdArg&, const RooCmdArg&, const RooCmdArg&, const RooCmdArg&, const RooCmdArg&, const RooCmdArg&)"
     )
     def fitTo(self, *args, **kwargs):
-        """This function is pythonized with the command argument pythonization.
-        The keywords must correspond to the CmdArgs of the RooAbsPdf::fitTo() function.
+        """The RooAbsPdf::fitTo() function is pythonized with the command argument pythonization.
+        The keywords must correspond to the CmdArgs of the function.
         """
+        # Redefinition of `RooAbsPdf.fitTo` for keyword arguments.
         args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
         return self._fitTo(*args, **kwargs)
 
+    @cpp_signature(
+        "RooPlot *RooAbsPdf::plotOn(RooPlot* frame,"
+        "    const RooCmdArg& arg1=RooCmdArg::none(), const RooCmdArg& arg2=RooCmdArg::none(),"
+        "    const RooCmdArg& arg3=RooCmdArg::none(), const RooCmdArg& arg4=RooCmdArg::none(),"
+        "    const RooCmdArg& arg5=RooCmdArg::none(), const RooCmdArg& arg6=RooCmdArg::none(),"
+        "    const RooCmdArg& arg7=RooCmdArg::none(), const RooCmdArg& arg8=RooCmdArg::none(),"
+        "    const RooCmdArg& arg9=RooCmdArg::none(), const RooCmdArg& arg10=RooCmdArg::none()"
+        ");"
```

Comment:
```suggestion
        ") const;"
```
To make the doxygen autolink work correctly here.